Orange/misc/flagged_data.py

```diff
@@ -0,0 +1,44 @@
+import re
+import numpy as np
+from Orange.data import Table, Domain, DiscreteVariable
+
+FLAGGED_SIGNAL_NAME = "Flagged Data"
```

Comment:
Thinking about it again, I started liking the constant because it will indeed make us stick to the same name in all widgets.

The name is not perfect, though. Not only will we change "flagged" to something else, but it also suggests that it is a name of a flagged signal. `ANNOTATED_DATA_SIGNAL_NAME = "Data"` is better (than `ANNOTATED_SIGNAL_NAME`) but awfully long. Think about it...